File: 백준/Bronze/15449. Art/Art.py
l = list(map(int, input().split()))
l.sort()
count = 0
for i in range(len(l)):
    for j in range (i + 1, len(l)):
        for k in range(j + 1, len(l)):
            if l[k] < l[i] + l[j]:
                count += 1
print(count)

File: 백준/Bronze/15449. Art/Art.js
const l = require('fs').readFileSync('/dev/stdin').toString().split(' ').map(Number).sort((a, b) => a - b);
const len = l.length;
let count = 0;
for(let i = 0; i < len; i++){
    for(let j = i + 1; j < len; j++){
        for(let k = j + 1; k < len; k++){
            if(l[k] < l[i] + l[j]) count++;
        }
    }
}
console.log(count);